src/coordinator/roles.spec.js

```diff
@@ -161,121 +161,129 @@ describe('Coordinator/Roles', function() {
+        const roleProject = fixture.build('contxtRoleProject');
 
         const roles = new Roles(baseSdk, baseRequest, expectedHost);
-        const promise = roles.addStack(null, stack.id, roleStack.accessType);
+        const promise = roles.addProject(
+          null,
+          project.slug,
+          roleProject.accessType
+        );
 
         return expect(promise).to.be.rejectedWith(
-          'A roleId is required for adding a stack to a role.'
+          'A roleId is required for adding a project to a role.'
         );
       });
     });
 
-    context('when the stack ID is not provided', function() {
+    context('when the project slug is not provided', function() {
       it('throws an error', function() {
         const role = fixture.build('contxtRole');
-        const roleStack = fixture.build('contxtRoleStack');
+        const roleProject = fixture.build('contxtRoleProject');
         const roles = new Roles(baseSdk, baseRequest, expectedHost);
-        const promise = roles.addStack(role.id, null, roleStack.accessType);
+        const promise = roles.addProject(role.id, null, roleProject.accessType);
 
         return expect(promise).to.be.rejectedWith(
-          'A stackId is required for adding a stack to a role.'
+          'A projectSlug is required for adding a project to a role.'
         );
       });
     });
 
     context('when the access type is not provided', function() {
       it('throws an error', function() {
         const role = fixture.build('contxtRole');
-        const stack = fixture.build('contxtStack');
+        const project = fixture.build('contxtProject');
 
         const roles = new Roles(baseSdk, baseRequest, expectedHost);
-        const promise = roles.addStack(role.id, stack.id, null);
+        const promise = roles.addProject(role.id, project.slug, null);
 
         return expect(promise).to.be.rejectedWith(
-          'An accessType of "reader", "collaborator", or "owner" is required for adding a stack to a role.'
+          'An accessType of "reader", "collaborator", or "owner" is required for adding a project to a role.'
         );
       });
     });
 
     context('when the access type is not a valid value', function() {
       it('throws an error', function() {
         const role = fixture.build('contxtRole');
-        const stack = fixture.build('contxtStack');
+        const project = fixture.build('contxtProject');
 
         const roles = new Roles(baseSdk, baseRequest, expectedHost);
-        const promise = roles.addStack(role.id, stack.id, faker.random.word());
+        const promise = roles.addProject(
+          role.id,
+          project.slug,
+          faker.random.word()
+        );
 
         return expect(promise).to.be.rejectedWith(
-          'An accessType of "reader", "collaborator", or "owner" is required for adding a stack to a role.'
+          'An accessType of "reader", "collaborator", or "owner" is required for adding a project to a role.'
```

Comment:
was the work to remove `collaborator` in another ticket? I remember talking about it, but can't remember which ticket. Either way, should we include that change here?

Reply:
I know we talked about it as well. I'm not sure what ticket its captured in so i"ll just remove it here to be pre-emtpive